# Provenance: TrebleDroid ci-20230706; original patch authors: Pierre-Hugues Husson.
# See ../ATTRIBUTION.md. This import is not authored by yukino1111.
diff --git a/src/adaptation/nfc_config.cc b/src/adaptation/nfc_config.cc
index 264968e488c63cce14f6c8f01b86d381d22c9193..92606979baf32d5fdc1ed93c98216888c6780a6b 100644
--- a/src/adaptation/nfc_config.cc
+++ b/src/adaptation/nfc_config.cc
@@ -30,7 +30,7 @@ using namespace ::android::base;
 namespace {
 std::string searchConfigPath(std::string file_name) {
   const std::vector<std::string> search_path = {
-      "/product/etc/", "/odm/etc/", "/vendor/etc/", "/system_ext/etc/", "/etc/",
+      "/odm/etc/", "/vendor/etc/", "/product/etc/", "/system_ext/etc/", "/etc/",
   };
   for (std::string path : search_path) {
     path.append(file_name);
